Adds integration docs with SuperTokens (#6888)
supabase/supabase: web/docs/guides/integrations/supertokens.mdx

---
id: supertokens
title: 'SuperTokens'

> description: "Create a Next.js application secured by Email-Password or Social Login from Stytch, and Supabase's Row Level Security (RLS)."

Reviewer: ```suggestion
description: "Create a Next.js application secured by SuperTokens and PostgreSQL Row Level Security."
```

Click `Review` and then `Save policy`.

### INSERT query policy

Our second policy will check whether the `user_id` being inserted is the same as the `userId` in the JWT.

Create another policy and add the following:

![Policy settings for INSERT](/img/guides/integrations/supertokens/policy_config_insert.png)

Similar to the previous policy we are calling the PostgreSQL function we created to get the currently logged in user's ID `auth.user_id()` and check whether this matches the `user_id` column for the row we are trying to insert. If it does, then it will allow the user to insert the row, otherwise it will continue to deny.

Click `Review` and then `Save policy`.

## Step 9: Test your changes

You can now sign up and you should see the following screen:

![SuperTokens App Authenticated](/img/guides/integrations/supertokens/supabase_app_authenticated_screen.png)

If you navigate to your table you should see a new row with the user's `user_id` and `email`.

![Supabase Users table](/img/guides/integrations/supertokens/table_with_user.png)

## Resources

- [SuperTokens](https://supertokens.com/) official website.

> - [SuperTokens blog](https://supertokens.com/blog).

Reviewer: ```suggestion
```

I would like to remove the link to the blog - it doesn't directly help with the tutorial above
